### PR TITLE
#14 create sidebar for tools

### LINKED ISSUE #14: Finish sidebar
need to feat the design.
check if it is possible to make the links dynamic from tools.

*the sidebar need to be fixed position on the scroll.

### PR DESCRIPTION
**Pull request by nemo369**

### CODE REVIEW

File: components/tools/ToolsSideBar.vue [hunk 1]
```diff
@@ -1,30 +1,98 @@
...
-    menus: [
-      { name: 'שאילתות ונאומים בני דקה', link: 'one-minute-speech' },
-      { name: 'הצעות לסדר יום', link: 'genreal-agenda' },
-    ],
+    isClose: false,
   }),
+  computed: {
+    routes() {
+      const toolRoots = this.$router.options.routes.find(route => route.path === '/tools');
+      if (!toolRoots && !toolRoots.children) {
```
Comment (YochayCO) [inline components/tools/ToolsSideBar.vue:36]: I believe it should be `||` instead `&&`

File: components/tools/ToolsSideBar.vue [hunk 2]
```diff
@@ -1,30 +1,98 @@
...
+        .filter(route => !!route.path)
+        .map(route => ({
+          ...route,
+          text: getRouteName(route.path),
+          isActive: this.isActive(route.path)
+        }));
+    }
+  },
+  methods: {
+    toglleSideBar() {
```
Comment (YochayCO) [inline components/tools/ToolsSideBar.vue:49]: toggle (it's in other places too)

**Review by YochayCO — COMMENTED** [review]
A bit late for me, I guess it's better than nothing

File: components/tools/ToolsSideBar.vue (hunk 1 above)
Comment (nemo369) [reply components/tools/ToolsSideBar.vue:36]: Thank you I will fix it

**Pull request merged**